values/secret.go

```diff
@@ -0,0 +1,20 @@
+package values
```

Comment:
Why does the `values` package need to know about secrets? I think these helper methods should exist in the `secrets` package itself.

Reply:
My intention was that the code to access secrets should be in a common location that's abstracted away from influxdb, but I'm not sure that's actually needed so I'll move this code to the `secrets` package.